css/main.css

```
@@ -0,0 +1,138 @@
+* {
+    box-sizing: border-box;
+    margin: 0;
+    outline: 0;
+    padding: 0;
+    position: relative;
+}
+
+.gallery {
+    display: flex;
+    justify-content: center;
+}
+
+.gallery .gallery__panel {
+    background-color: black;
+    display: flex;
+    justify-content: center;
+    padding: 10px;
+    position: fixed;
+    top: 0;
+    width: 100%;
+    z-index: 1;
+}
+
+.gallery .gallery__container {
+    display: flex;
+    flex-wrap: wrap;
+    border: 1px solid black;
+    margin-top: 65px;
+    max-width: 1062px;
+    padding: 5px;
+}
+
+.gallery__panel button {
+    padding: 10px 14px;
+    background-color: white;
+    color: black;
+    border: 1px solid black;
+    border-radius: 5px;
+    margin-left: 10px;
+    transition: background-color 0.5s;
+    cursor: pointer;
+}
+
+.gallery__panel input {
+    padding-top: 0;
+    padding-right: 8px;
+    padding-bottom: 0;
+    padding-left: 8px;
+}
+
+.gallery .gallery__cell {
+    display: flex;
+    flex-direction: column;
+    margin: 5px 5px 5px 5px;
```

```suggestion
    margin: 5px;
```
Ponieważ jest krótsze i czytelniejsze.